### PR TITLE
[Design2-93] Design System Checkbox - Update/fix vertical alignment of checkbox and it's label

### PR DESCRIPTION
**Pull request by levadadenys**
### [Design2-93] Design System Checkbox - Update/fix vertical alignment of checkbox and it's label
After the font change we have a slight misalignment of checkbox content, this PR fixes it.

Before:
Знімок екрана 2024-02-02 о 18 23 15

After:
Знімок екрана 2024-02-02 о 18 23 06

## Links
- jira ticket: [DESIGN2-93](https://codedotorg.atlassian.net/browse/DESIGN2-93)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/componentLibrary/checkbox/checkbox.module.scss [hunk 1]
```diff
@@ -20,7 +20,6 @@
 
   span {
     margin-bottom: 0;
-    line-height: initial;
```
Comment (fisher-alice) [inline apps/src/componentLibrary/checkbox/checkbox.module.scss:23]: I see removing this line seems to increase the `line-height`. Just curious why you included this change. Also, where is this`initial` value defined? Thanks!

**Review by fisher-alice — APPROVED** [review]
LGTM! Just left a question out of curiosity. Thanks!

File: apps/src/componentLibrary/checkbox/checkbox.module.scss (hunk 1 above)
Comment (levadadenys) [reply apps/src/componentLibrary/checkbox/checkbox.module.scss:23]: Yep, it increases the line height to be what initially it should've be :D. Reason I added this `initial` value was to fix vertical alignment for previous font we were using.

Styles for this span are set in tsx part, we're using `Typography` `bodyText-${size}` styles here (they have line height)

Comment (fisher-alice) [reply apps/src/componentLibrary/checkbox/checkbox.module.scss:23]: Thanks for the explanation!

**Review by sanchitmalhotra126 — APPROVED** [review]

**Pull request merged**